### PR TITLE
Added command parameter to user stopped typing behavior

### LINKED ISSUE #914: Add CommandParameter to UserStoppedTypingBehavior
### Description
```
<Entry>
    <Entry.Behaviors>
        <xct:UserStoppedTypingBehavior 
            Command="{Binding SearchCommand}"
            StoppedTypingTimeThreshold="1000"
            MinimumLengthThreshold="3"
            ShouldDismissKeyboardAutomatically="True" />
    </Entry.Behaviors>
</Entry>
```

If I use this code inside a CollectionView, how can I pass the object to the Command? Is CommandParameter missing?
### Steps to Reproduce

Use a CollectionView with an entry as ItemTemplate

```
<Entry
                                    Grid.Column="3"
                                    Style="{StaticResource NumeriEntryStyle}"
                                    Text="{Binding FasciCaricati}">
                                    <Entry.Behaviors>
                                        <xct:UserStoppedTypingBehavior
                                            Command="{Binding Path=BindingContext.CalcolaTonnellateCaricateCommand, Source={x:Reference Name=this}}"
                                            MinimumLengthThreshold="1"
                                            ShouldDismissKeyboardAutomatically="True"
                                            StoppedTypingTimeThreshold="2000" />
                                    </Entry.Behaviors>
                                </Entry>
```

### Expected Behavior
I would like to pass the object I am setting the property to the Command
### Actual Behavior
I don't know how
### Basic Information

- Version with issue: last nightly 
- Last known good version: unknown
- IDE: Visual Studio 2019 16.8.5
- Platform Target Frameworks: 
  - iOS:  
  - Android:  10
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

### PR DESCRIPTION
**Pull request by jBijsterboschNL**
### Description of Change ###

Added `CommandParameter` bindable property to the existing `UserStoppedTypingBehavior`. When this CommandParameter is specified, it will be used when executing the specified Command, otherwise the `InputView.Text` value is used as the `CommandParameter`.

### Bugs Fixed ###

- Fixes #914 

### API Changes ###

Added: 
 
- `CommandParameter` bindable property to `UserStoppedTypingBehavior`

### Behavioral Changes ###

When this new CommandParameter is specified, it will be used when executing the specified Command, otherwise the `InputView.Text` value is used as the `CommandParameter`.

### PR Checklist ###
- [x] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [x] Changes adhere to coding standard
- [ ] Updated [documentation](https://github.com/MicrosoftDocs/xamarin-communitytoolkit)

### CODE REVIEW

**Comment by jBijsterboschNL** [comment]
I've marked this PR as a draft for now as there is still one question about the exact working of this new `CommandParameter` property on the `UserStoppedTypingBehavior`. For now I've implemented it as described above.

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs [hunk 1]
```diff
@@ -18,6 +18,12 @@ public class UserStoppedTypingBehavior : BaseBehavior<InputView>
 		public static readonly BindableProperty CommandProperty
 			= BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(UserStoppedTypingBehavior));
 
+		/// <summary>
+		/// Backing BindableProperty for the <see cref="CommandParameter"/> property.
+		/// </summary>
```
Comment (jfversluis) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs:23]: 😍

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs [hunk 2]
```diff
@@ -47,6 +53,12 @@ public ICommand Command
 			set => SetValue(CommandProperty, value);
 		}
 
+		public object CommandParameter
```
Comment (jfversluis) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs:56]:
```suggestion
		/// <summary>
		/// An optional parameter to forward to the <see cref="Command"/>. This is a bindable property.
		/// </summary>
		public object CommandParameter
```

**Review by jfversluis — CHANGES_REQUESTED** [review]

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs (hunk 1 above)
Comment (acaliaro) [reply src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs:23]: @jfversluis will be in the next nightly?

Comment (jfversluis) [reply src/CommunityToolkit/Xamarin.CommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs:23]: It's not merged yet so no

**Review by jfversluis — APPROVED** [review]

**Comment by acaliaro** [comment]
@jfversluis seems to be merged but I don't see it in the nightly build

**Comment by jfversluis** [comment]
Will be tomorrow

**Pull request merged**